ProcessLib/HT/HTFEM.h

```diff
@@ -251,14 +255,60 @@ class LocalAssemblerData : public HTLocalAssemblerInterface
                  N.transpose() * velocity.transpose() * dNdx *
                      fluid_reference_density * specific_heat_capacity_fluid) *
                 w;
-            Kpp.noalias() += w * dNdx.transpose() * perm_over_visc * dNdx;
+            Kpp.noalias() += w * dNdx.transpose() * K_over_mu * dNdx;
             Mtt.noalias() += w * N.transpose() * heat_capacity * N;
             Mpp.noalias() += w * N.transpose() * specific_storage * N;
```

Comment:
To enable thermal expansion, Mpt is needed.
Mpt.noalias() += w * N.transpose() * eff_thermal_expansion  * N;
where
```
const double eff_thermal_expansion =
            3.0 * (biot_constant - porosity) * solid_thermal_expansion -
porosity * drho_dT(p, T) / rho;
```
See https://github.com/ufz/ogs/blob/master/ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-impl.h#L235